samples/apps/forum/src/controllers/site.ts

```diff
@@ -34,6 +34,10 @@ const HEADER_HTML = `
         padding: 3rem 1.5rem;
         text-align: center;
       }
+      #login-btn {
+          display: none;
+          margin-left: 15px;
```

Comment:
Since we're using rem/em, I suggest we do here too rather than px :)

Reply:
Done.